src/app/authorization/login/login.component.scss

```diff
@@ -1,12 +1,16 @@
-*{
-    font-family: Roboto;
-}
-h6{
-    margin-top: 14px;
-}
-// TODO:Please do not apply on direct class of boostrap 
-.card{
-    margin: 0 auto; 
-    border-radius: 15px;
-    height: auto;    
-}
\ No newline at end of file
+.image {
+  width: 4rem;
+  height: 4rem;
+}
+label {
+  font-size: calc(70% + 1vh);
+}
```

Comment:
please use class name not HTML tag directly until unless it is necessary

Reply:
This change is in upcoming PR